logic/transaction.js

```diff
@@ -425,6 +426,12 @@ class Transaction {
 			return setImmediate(cb, 'Multisig request is not allowed');
 		}
 
+		// Check if sender account has second signature enabled.
+		// Abort if registering again.
+		if (transaction.type === transactionTypes.SIGNATURE && sender.secondSignature) {
+			return setImmediate(cb, 'This account is already enabled with second signature');
```

Comment:
Maybe it's better to have a similar error message wording as the rest of errors.
```suggestion
			return setImmediate(cb, 'Sender already has second signature enabled');
```

Reply:
Makes sense, updated :)